EventFilter/CTPPSRawToDigi/python/ctppsRawToDigi_cff.py

```diff
@@ -18,13 +18,13 @@
     ),
     # during TS2 (2016)
     cms.PSet(
-      validityRange = cms.EventRange("280386:min - 281600:max"),
+      validityRange = cms.EventRange("280385:min - 281600:max"),
       mappingFileNames = cms.vstring(),
       maskFileNames = cms.vstring()
     ),
     # after TS2 (2016)
     cms.PSet(
-      validityRange = cms.EventRange("281601:min - 999999999:max"),
+      validityRange = cms.EventRange("281600:min - 999999999:max"),
```

Comment:
Now I'm missing how this code works: the previous range ends at 281600:max, while this range starts at 281600:min, which at least semantically suggests there is an overlap.
Please explain why there is no overlap and see if it's possible to remove it even in config semantics.